custom_components/ramses_cc/broker.py

```diff
@@ -276,37 +276,38 @@ async def async_add_entities(
             )
 
         def find_new_entities(
-            known: list[RamsesRFEntity], current: list[RamsesRFEntity]
-        ) -> tuple[list[RamsesRFEntity], list[RamsesRFEntity]]:
-            new = [x for x in current if x not in known]
-            return known + new, new
+            known: set[RamsesRFEntity], current: set[RamsesRFEntity]
+        ) -> set[RamsesRFEntity]:
+            new = current - known
+            known |= new
+            return new
```

Comment:
This makes me nervous. Why did we have the tuple before - why was it needed then, and is no longer needed now?